client/app/components/land-use-action.hbs

```diff
@@ -29,17 +29,17 @@
           <div class="shrink cell">
             <label for="middle-label" class="middle">How many {{landUseAction.name}} actions?</label>
           </div>
-        </div>
-        <div class="auto cell">
-          <ActionFormInput 
-            @pasForm={{@pasForm}}
-            @fieldType="number"
-            @length="10"
-            @landUseActionField={{landUseAction.countField}}
-          />
-          <ValidationMessage
-            @changesetError={{get @submittableChanges.error landUseAction.countField}}
-          />
+          <div class="auto cell">
+            <ActionFormInput 
+              @pasForm={{@pasForm}}
+              @fieldType="number"
+              @length="10"
+              @landUseActionField={{landUseAction.countField}}
+            />
+            <ValidationMessage
+              @changesetError={{get @submittableChanges.error landUseAction.countField}}
+            />
+          </div>
```

Comment:
Hmm do you see the spacing issues in the diff here? It doesn't even look like anything was added. How come there are new spaces added?

Reply:
The ActionFormInput was moved inside another div that was previously above it

Reply:
It's hard to see on the git diff :/ 

<img width="1357" alt="Screen Shot 2020-05-19 at 12 15 41 PM" src="https://user-images.githubusercontent.com/26672885/82357293-83afc700-99ca-11ea-87c5-344bfedcff89.png">

Reply:
Oh sorry I think my Sourcegraph plugin is going haywire 
![image](https://user-images.githubusercontent.com/5004319/82360139-ed35d280-99d6-11ea-9ec1-0ff848ab98ea.png)

Reply:
@allthesignals you can hide whitespace changes here:
![image](https://user-images.githubusercontent.com/5316367/82365980-751fda80-99df-11ea-94dc-7d95aeec1540.png)